drivers/volume/portworx/portworx.go

```diff
@@ -1037,6 +1037,53 @@ func (d *portworx) ExitMaintenance(n node.Node) error {
 	return nil
 }
 
+func (d *portworx) EnterPoolMaintenance(n node.Node) error {
+	cmd := fmt.Sprintf("pxctl sv pool maintenance -e -y")
+	out, err := d.nodeDriver.RunCommand(
+		n,
+		cmd,
+		node.ConnectionOpts{
+			Timeout:         maintenanceWaitTimeout,
+			TimeBeforeRetry: defaultRetryInterval,
+		})
+	if err != nil {
+		log.Errorf(fmt.Sprintf("error when entering pool maintenance, Err: %v", err))
+		return err
+	}
+	log.Infof("Enter pool maintenance %s", out)
+	return nil
+}
+
+func (d *portworx) ExitPoolMaintenance(n node.Node) error {
+	cmd := fmt.Sprintf("pxctl sv pool maintenance -x -y")
+	out, err := d.nodeDriver.RunCommand(
+		n,
+		cmd,
+		node.ConnectionOpts{
+			Timeout:         maintenanceWaitTimeout,
+			TimeBeforeRetry: defaultRetryInterval,
+		})
+	if err != nil {
+		log.Errorf(fmt.Sprintf("error when exiting pool maintenance, Err: %v", err))
```

Comment:
Don't `log.Errorf()`, just return and lets add a node name, will be easier to debug
`return fmt.Errorf("error when exiting pool maintenance on node [%s], Err: %v", n.Name, err)

Reply:
fixed